samples/transfer-workflow-orchestration/src/main/java/com/example/transfer/application/WalletService.java

```diff
@@ -0,0 +1,29 @@
+package com.example.transfer.application;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class WalletService {
```

Comment:
Maybe good to add a note here saying that this is a dummy wallet service and is just simulating a real one.

Reply:
and that the real service could be an external service that is called with HTTP